primitives/src/util/tests/prep_db.rs

```diff
@@ -5,30 +5,30 @@ use lazy_static::lazy_static;
 use std::collections::HashMap;
 
 lazy_static! {
-    pub static ref IDS: HashMap<&'static str, String> = {
+    pub static ref IDS: HashMap<String, String> = {
         let mut ids = HashMap::new();
 
-        ids.insert("leader", "awesomeLeader".into());
-        ids.insert("follower", "awesomeFollower".into());
-        ids.insert("user", "awesomeTestUser".into());
-        ids.insert("publisher", "b7d3f81e857692d13e9d63b232a90f4a1793189e".into());
-        ids.insert("publisher2", "myAwesomePublisher2".into());
-        ids.insert("creator", "awesomeCreator".into());
-        ids.insert("tester", "2892f6C41E0718eeeDd49D98D648C789668cA67d".into());
+        ids.insert("leader".into(), "awesomeLeader".into());
+        ids.insert("follower".into(), "awesomeFollower".into());
+        ids.insert("user".into(), "awesomeTestUser".into());
+        ids.insert("publisher".into(), "b7d3f81e857692d13e9d63b232a90f4a1793189e".into());
+        ids.insert("publisher2".into(), "myAwesomePublisher2".into());
+        ids.insert("creator".into(), "awesomeCreator".into());
+        ids.insert("tester".into(), "2892f6C41E0718eeeDd49D98D648C789668cA67d".into());
 
         ids
     };
 
-    pub static ref AUTH: HashMap<&'static str, String> = {
+    pub static ref AUTH: HashMap<String, String> = {
```

Comment:
I've changed that as using `String` will make us type just a bit more when trying to pick a specific key:
`AUTH["my_key".to_string()]` or `AUTH["my_key".to_owned()]` instead of just doing `AUTH["my_key"]`

Reply:
hmnn .. I am not sure this is correct 
https://doc.rust-lang.org/std/collections/struct.HashMap.html#examples
```
// Check for a specific one.
// When collections store owned values (String), they can still be
// queried using references (&str).
```

Reply:
Haha, yeah, nice, hooray for rustlang again!